packages/shared/lib/clients/orchestrator.ts

```diff
@@ -121,7 +130,7 @@ export class Orchestrator {
         retryMax: number;
         async: boolean;
         logCtx: LogContext;
-    }): Promise<Result<{ id: string; statusUrl: string } | { data: T }, NangoError>> {
+    }): Promise<Result<AsyncActionResponse | { data: T }, NangoError>> {
```

Comment:
not related to webhooks but I was repeating the response type in a few different places. Created a type in types package instead